test/integration/replicationcontroller/replicationcontroller_test.go

```diff
@@ -24,24 +24,29 @@ import (
 	"time"
 
 	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/informers"
 	clientset "k8s.io/client-go/kubernetes"
+	typedv1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	restclient "k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/retry"
+	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
 	"k8s.io/kubernetes/pkg/controller/replication"
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
```

Comment:
At the time, I decided to leave it out because it made sense to me that a test of RC behavior should be sort of frozen in time, because RC is deprecated and we don't intend to add new behavior to it. Keeping the RC tests fixed could also help make sure we don't accidentally change guaranteed parts of RC behavior while adding new code to ReplicaSetController.

However, if the goal is for RC to keep evolving alongside RS, just with the old selector type, then it would make sense to ensure we update their tests in sync.